src/components/domain/project/projectList/LinkProjectList.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import BorderCard from '@/components/common/card/BorderCard';
+import { ComponentSpinner } from '@/components/common/spinner';
+import ProjectSummaryCard from '@/components/domain/project/projectCard/ProjectSummaryCard';
+import { useGetLinkProjectsByProjectName } from '@/hooks/queries/useProjectService';
+import { convertStringToDate } from '@/lib/dateTime';
+import { PROJECT_CARD_VARIANT, type ProjectSummaryData } from '@/models/project/projectModels';
+
+interface LinkProjectListProps {
+  searchProjectName: string;
+}
+export default function LinkProjectList({ searchProjectName }: LinkProjectListProps) {
+  const { data, isLoading, isError } = useGetLinkProjectsByProjectName(searchProjectName);
+
+  const projectList = data?.data;
```

Comment:
여기 `(project): ProjectSummaryData => ({ ... })` 이런 식으로  map 함수 내부의 반환 객체에 대한 타입이 명시적으로 있으면 좋을 것 같아요. 그리고 삼항 연산자보다 옵셔널 체이닝이 조금 더 간결할 것 같아요.
projectList를 따로 선언하는게 좋으시다면 projectDatas 맵핑 부분에 타입만 명확히 적어주시면 좋을 것 같습니다!

```suggestion
  const projectDatas: ProjectSummaryData[] =
    data?.data?.map(
      (project): ProjectSummaryData => ({
        projectId: project.projectId,
        projectname: project.projectName,
        organizationName: project.organizationName,
        startDate: convertStringToDate(project.startDate),
        endDate: convertStringToDate(project.endDate),
        categories: project.categories,
      }),
    ) || [];
```

Reply:
✅ 8f1fdc0bb34fbca4085e0d7328ca3a249415ed74